minifi/minifi-docs/src/main/markdown/minifi-java-agent-quick-start.md

```diff
@@ -91,6 +91,8 @@ This launches MiNiFi and leaves it running in the foreground. To shut down NiFi,
 When you are working with a MiNiFi dataflow, you should design it, add any additional configuration your environment or use case requires, and then deploy your dataflow. MiNiFi is not designed to accommodate substantial mid-dataflow configuration.
 
 ## Setting up Your Dataflow
+
+### Manually from a NiFi dataflow
```

Comment:
For case consistency, change to:

Manually From a NiFi Dataflow

Reply:
Applied